contracts/src/contracts/auction.py

```diff
@@ -64,6 +65,29 @@ def closeAccountTo(account: Expr) -> Expr:
             )
         )
 
+    @Subroutine(TealType.none)
+    def sendBidToSeller() -> Expr:
+        return Seq(
+            InnerTxnBuilder.Begin(),
+            InnerTxnBuilder.SetFields(
+                {
+                    TxnField.type_enum: TxnType.Payment,
+                    TxnField.receiver: App.globalGet(seller_key),
+                    TxnField.amount: (
+                        App.globalGet(lead_bid_amount_key)
+                        - (
+                            (
+                                App.globalGet(fee_percent_key)
+                                * App.globalGet(lead_bid_amount_key)
+                            )
+                            / Int(100)
+                        )
```

Comment:
Integer math is fun! This is the same as `bidAmount * (1 - (royaltyFee / 100))` where `royaltyFee` is the percentage express as a whole number from 0 to 100 (inclusive).